xml/security-troubleshooting_tls.xml

```diff
@@ -95,6 +95,94 @@ ansible-playbook -i hosts/verb_hosts tls-reconfigure.yml --limit TLS-CA
 <screen>ansible-playbook -i hosts/verb_hosts monasca-stop.yml
 ansible-playbook -i hosts/verb_hosts monasca-start.yml</screen>
  </section>
+ <section>
+  <title>Expired TLS Certificates</title>
+  <para>
+   Use the following steps to recreate expired TLS certificates for MySQL
+   Percona clusters.
+  </para>
+  <procedure>
+   <step>
+    <para>
+     Stop &mariadb; with a playbook
+    </para>
+    <substeps>
+     <step>
+       <screen>&prompt.ardana;cd ~/scratch/ansible/next/hos/ansible
+&prompt.ardana;ansible-playbook -i hosts/verb_hosts percona-stop.yml</screen>
+     </step>
+     <step>
+      <para>
+       If the playbook failed, use the <literal>systemd</literal> command to
+       shutdown the daemon on each database node.
+      </para>
+      <screen>&prompt.ardana;sudo systemctl stop mysql</screen>
+     </step>
+     <step>
+      <para>
+       If the <literal>mysql</literal> daemon does not go down following the
+       <command>systemctl stop</command>, then kill the daemon using
+       <command>kill -9 mysqld</command> before continuing.
+      </para>
+     </step>
+    </substeps>
+   </step>
+   <step>
+    <para>
+     Verify no mysqld processes are running. The following command should show
+     no entries.
+    </para>
+    <screen>&prompt.ardana;ps -aux | grep mysql</screen>
+   </step>
+   <step>
+    <para>
+     Commment out all lines with <literal>ssl</literal> in
+     <filename>/etc/mysql/my.cnf</filename> on all nodes.
+    </para>
+   </step>
+   <step>
+    <para>
+     Bootstrap the first controller node
+    </para>
+    <screen>&prompt.ardana;cd ~/scratch/ansible/next/ardana/ansible
+&prompt.ardana;ansible-playbook -i hosts/verb_hosts percona-bootstrap.yml</screen>
+   </step>
+   <step>
+    <para>
+     Re-create the SSL certificates
```

Comment:
Recreate or re-create? You use both.

Reply:
there is no recreate in security. It's all serious stuff. all are now re-create. Thank you.